spec/statement.dd

```diff
@@ -1128,6 +1128,7 @@ $(GNAME StatementNoCaseNoDefault):
 
         $(P A $(I CaseRangeStatement) is a shorthand for listing a series
         of case statements from $(I FirstExp) to $(I LastExp).
+        The maximum number of such case statements is 257.
```

Comment:
The maximum number of such case statements *per CaseRangeStatement*. 
This is accepted by dmd: `case 0: .. case 256: case 257: .. case 512:`

Reply:
I am pretty sure it's an off-by-one error in the check.
either it should be a power-of-two or not be limited at all.

Reply:
It is, see https://github.com/dlang/dmd/pull/13058
There is no reason the limit needs to be a power of 2, we could make it 300 if we want to. Lowering it would be a breaking change (although an unlikely one).

Reply:
Why have a limit in the specification at all (i.e. why actually pick a number beyond making an implementation choose one)?